### ISSUE TITLE
Unable to alias process.js

### ISSUE THREAD

**Issue by Ignavia** [jspm/jspm-cli#1601]
The following build fails on JSPM 0.17.0-beta.11:

```
> jspm build ./src/util.js - lodash ./dist/util.js --format cjs --skip-source-maps --minify

     Creating the single-file build for ./src/util.js - lodash...

err  Error: Unable to alias the conditional dependency "github:jspm/nodelibs-process@0.2.0-alpha/#:./process.js".
    at getAlias (c:\Users\larsr\Dropbox\Uni\semester_7\packages\util\node_modules\systemjs-builder\lib\utils.js:73:11)
    at c:\Users\larsr\Dropbox\Uni\semester_7\packages\util\node_modules\systemjs-builder\lib\compile.js:417:20
    at Array.map (native)
    at c:\Users\larsr\Dropbox\Uni\semester_7\packages\util\node_modules\systemjs-builder\lib\compile.js:414:26
    at tryCatcher (c:\Users\larsr\Dropbox\Uni\semester_7\packages\util\node_modules\bluebird\js\release\util.js:16:23)
    at Promise._settlePromiseFromHandler (c:\Users\larsr\Dropbox\Uni\semester_7\packages\util\node_modules\bluebird\js\release\promise.js:503:31)
    at Promise._settlePromise (c:\Users\larsr\Dropbox\Uni\semester_7\packages\util\node_modules\bluebird\js\release\promise.js:560:18)
    at Promise._settlePromise0 (c:\Users\larsr\Dropbox\Uni\semester_7\packages\util\node_modules\bluebird\js\release\promise.js:605:10)
    at Promise._settlePromises (c:\Users\larsr\Dropbox\Uni\semester_7\packages\util\node_modules\bluebird\js\release\promise.js:684:18)
    at Promise._fulfill (c:\Users\larsr\Dropbox\Uni\semester_7\packages\util\node_modules\bluebird\js\release\promise.js:629:18)
    at c:\Users\larsr\Dropbox\Uni\semester_7\packages\util\node_modules\bluebird\js\release\nodeback.js:42:21
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:404:3)
```

If I just run `jspm build ./src/util.js ./dist/util.js --format cjs --skip-source-maps --minify` it works. The same error occurs if I subtract any other package from ./src/util.js.

**package.json:**

```
...
  "jspmPackage": true,
  "registry": "npm",
  "format": "cjs",
  "jspm": {
    "name": "@ignavia/util",
    "dependencies": {
      "lodash": "^4.6.1"
    },
    "devDependencies": {
      "babel-regenerator-runtime": "^6.5.0",
      "core-js": "^2.1.4",
      "plugin-babel": "systemjs-plugin-babel@^0.0.8"
    },
    "peerDependencies": {
      "fs": "github:jspm/nodelibs-fs@^0.2.0-alpha",
      "path": "github:jspm/nodelibs-path@^0.2.0-alpha",
      "process": "github:jspm/nodelibs-process@^0.2.0-alpha"
    }
  } ...
```

**jspm.config.js:**
```
SystemJS.config({
  transpiler: "plugin-babel",
  packages: {
    "@ignavia/util": {
      "main": "util.js",
      "format": "esm",
      "defaultExtension": "js",
      "meta": {
        "*.js": {
          "loader": "plugin-babel"
        }
      }
    }
  }
});

SystemJS.config({
  packageConfigPaths: [
    "npm:@*/*.json",
    "npm:*.json",
    "github:*/*.json"
  ],
  map: {
    "babel-regenerator-runtime": "npm:babel-regenerator-runtime@6.5.0",
    "core-js": "npm:core-js@2.1.4",
    "fs": "github:jspm/nodelibs-fs@0.2.0-alpha",
    "lodash": "npm:lodash@4.6.1",
    "path": "github:jspm/nodelibs-path@0.2.0-alpha",
    "plugin-babel": "npm:systemjs-plugin-babel@0.0.8",
    "process": "github:jspm/nodelibs-process@0.2.0-alpha"
  },
  packages: {}
});
```

**Comment by guybedford** [comment]
Thanks for sharing this case, yes that's a bug - fixed in https://github.com/systemjs/builder/commit/a015a0d16fba8f46a95c723931674459e241671a.

**Comment by guybedford** [comment]
Fixed in jspm 0.17.0-beta.12.
